AppleStore/AppleStore/AboutGoodViewController.swift

```diff
@@ -28,14 +28,20 @@ final class AboutGoodViewController: UIViewController {
     private var picOfDeliveryImageView = UIImageView()
     private var picsScrollView = UIScrollView()
     
+    // MARK: - Private Properties
+    
+    var tapFirstPic = UITapGestureRecognizer()
+    var tapSecondPic = UITapGestureRecognizer()
+    var tapThirdPic = UITapGestureRecognizer()
     
     // MARK: - Public Properties
     
     var picOne = String()
     var picTwo = String()
     var picThree = String()
     var name = String()
-    var price =  String()
+    var price = String()
+    var urlOfGood = String()
```

Comment:
```suggestion
    var goodUrlString = String()
```
of по гайду не пишим нигде